omise/controllers/front/payment.php

```diff
@@ -0,0 +1,28 @@
+<?php
+if (! defined('_PS_VERSION_')) {
+    exit();
+}
+
+if (defined('_PS_MODULE_DIR_')) {
+    require_once _PS_MODULE_DIR_ . '/omise/classes/charge.php';
+}
+
```

Comment:
@nimid how about try to think in MVC pattern?

Assumed, 
- `OmisePaymentModuleFrontController` = Controller
- `Charge` = Model
- `OmiseCharge` = DB

Should model care about how to retrieve input by itself (by retrieving `$_POST['card']` inside a model) or better to pass through controller?